src/screens/Drawer/TaskDetailsScreen.tsx

```diff
@@ -21,6 +24,7 @@ export const TaskDetailsScreen = (props) => {
 
     const closeModalHandler = () => {
         setModalVisible(false);
+        dispatch(resetSwapConfig());
```

Comment:
no need for loading logic?

Reply:
no, this is just to clear reducer of any data from user actions on modal close